py/selenium/webdriver/common/service.py

```diff
@@ -160,14 +160,9 @@ def stop(self):
                     except AttributeError:
                         pass
                 self.process.terminate()
-                self.process.kill()
                 self.process.wait()
+                self.process.kill()
                 self.process = None
         except OSError:
             # kill may not be available under windows environment
             pass
-
-    def __del__(self):
-        # subprocess.Popen doesn't send signal on __del__;
-        # we have to try to stop the launched process.
-        self.stop()
```

Comment:
can we add a try / except around this to make it more stable?

I like the idea of any mistakenly un-quit drivers closing down after the process is quit (the java server does this... also i forget to do driver.quit often when i use the command line repl :) )